front/lib/plans/pro_plans.ts

```diff
@@ -18,22 +18,21 @@ export type PlanAttributes = Omit<
  */
 
 // Current pro plans:
-export const PRO_PLAN_MAU_29_CODE = "PRO_PLAN_MAU_29";
-export const PRO_PLAN_FIXED_1000_CODE = "PRO_PLAN_FIXED_1000";
+export const PRO_PLAN_CODE = "PRO_PLAN_SEAT_29";
 
 /**
  * Paid plans are stored in the database.
  * We can update existing plans or add new one but never remove anything from this list.
  * Entreprise custom plans will be created from Poké.
  */
 
-const PRO_PLANS_DATA: PlanAttributes[] = [
-  {
-    code: "PRO_PLAN_SEAT_29",
+const PRO_PLANS_DATA: PlanAttributes[] = [];
+
+if (isDevelopment()) {
+  PRO_PLANS_DATA.push({
+    code: PRO_PLAN_CODE,
     name: "Pro",
-    stripeProductId: isDevelopment()
-      ? "prod_OvrzyxfSDz5Jqd" // Pro plan in Stripe Test env
-      : "prod_OvrkkwZwLUOlpx", // Pro plan in Stripe Prod env
+    stripeProductId: "prod_OwKvN4XrUwFw5a",
```

Comment:
shouldn't this be in the env ?

Reply:
Fine with me being here since it's more convenient and not secret

Reply:
> Fine with me being here since it's more convenient and not secret

It's actually less convenient if we need to change it (but fine with me to have it hardcoded ofc)